calibration/loader.py

```diff
@@ -0,0 +1,123 @@
+import os
```

Comment:
This is almost a copy of `aspics/loader.py`. You added a bunch of new parameters to `aspics/loader.py` but not this one.

I think we should have one version of this file. It's going to be a nightmare to maintain two slowly diverging copies. I'll look into this...